app/helpers/koi/navigation_helper.rb

```diff
@@ -27,16 +27,14 @@ def get_nav_items(key)
       Rails.cache.fetch(prefix_cache_key(key), expires_in: cache_expiry) do
         NavItem.navigation(key, binding())
       end
+    elsif Koi::Caching.nav_memory_caching
+      @get_nav_items ||= {}
+      @get_nav_items[key] ||= NavItem.navigation(key, binding())
```

Comment:
as discussed, this cache is not valuable because we can use Koi caching or Rails query caching, no need for the controller-scoped caching

Reply:
removed this cache of navigation options